### ISSUE TITLE
No alerts -  traffic lights enforcement cameras (not mix with speed cameras enforcements)

### ISSUE THREAD

**Issue by Max1234-Ita** [osmandapp/OsmAnd#19641]
### Description

While speed camera alert works quite well (provided the camera belongs to an _enforcement=speed_ relation) when approaching traffic lights enforcement cameras, no alert is displayed nor voice alert is given, even though the speed_camera belongs to an enforcement relation (_enforcement=traffic_signals_).

I' ve tried several times, in the past days, with voice alerts enabled, with just screen alerts, with both of them enabled: I've driven by numerous traffic lights where enforcement cameras are installed, but I never saw an alert from OsmAnd; I've also encountered several speed cameras and got the corresponding alert for those which are properly mapped with an enforcement=maxspeed relation.

So, I guess the _enforcement=traffic_signals_ relation is disregarded by OsmAnd... which is actually a pity since, when driving in a city, being alerted about traffic signal enforcement is as important as being warned about speed checks: hope this can be implemented in the app.

I'll provide one as an example: https://www.openstreetmap.org/relation/13746143
In that case, I would expect a speed camera (more properly, traffic lights camera) alert to be issued when coming from West along the "Corso Ventisette Marzo" and approaching the crossing with "Via Papa Giovanni": the mapping of the enforcement relation seems to be OK.

<br>

### Attachments:
__
Short screen recording taken on the mentioned spot. I would have expected to see an alert popping up, i.e. just after crossing the "Via Piave" (video time: about 00:23)
https://github.com/osmandapp/OsmAnd/assets/15315381/10ba9510-f940-4293-97bb-da3c1652d8e7

__
Setting used while taking the video above (voice prompts were disabled at all, in that moment).

![Profile setup - Driving](https://github.com/osmandapp/OsmAnd/assets/15315381/6a48c877-654b-4e69-a578-fe43153816fd)

To easily find the spot used in my test just search for 'Voghera corso ventisette marzo' (Europe-Italy-Lombardy map required)

### Steps to reproduce

1. Launch OsmAnd; 
2. Make sure that at least a local vector map is installed (used in the test: Europe-Italy-Lombardy) and that rendering Engine v2 is enabled;
3. Set a destination and start navigating
4. Drive by a speed camera mapped in OSM with a relation of _type=enforcement_; _enforcement=maxspeed_; observe that an alert is issued to warn of the speed check;
5. Drive through a crossing where a traffic light camera is installed and mapped with a relation _type=enforcement_; _enforcement=traffic_signals_.

### Actual result

No alert is issued in the second case, even though the traffic light camera is properly mapped in OSM.

### Expected result

- A warning should be displayed if screen alerts are enabled for speed cameras (perhaps a dedicated option would be optimal solution, but let's not complicate things too much)
- A voice prompt should be issued if voice prompt alerts are enabled for speed cameras (see above) 

### Your Environment (required)

**WARNING** Crash-Logs **MAY** contain information you deem sensitive.
Review this **CAREFULLY** before posting your issue!

```
OsmAnd Version: OsmAnd Plus v.4.7.9 (stable)
Android/iOS version: Android 11
Device model: Samsung Galaxy A50 (SM-505 FN)
Crash-Logs: None
```

**Comment by zipav** [comment]
OsmAnd Nightly 4.8.0#2216m, released: 2024-04-30

Reproduced on the latest build of OsmAnd Nightly. Missing both Screen Alert and the voice prompt that warns of Speed cameras.

https://github.com/osmandapp/OsmAnd/assets/123936521/10d937de-758e-4d2f-a039-d7dff8700559

**Comment by Max1234-Ita** [comment]
> I don't think it's a speed camera - it's a traffic enforcement https://www.openstreetmap.org/relation/13746143#map=18/44.99235/9.00229. So I think it's purposefully not supported to not mix up

Yes, probably the mapping there needs to be reviewed: I'm a mapper since 2012, but I still have a lot to learn!

Anyway, being prompted by OsmAnd about enforcements (not only for speed but also for traffic lights, prohibited lanes, etc) would be a very useful feature to have... and perhaps it would save us some fine :-)
